saagieapi/projects/projects.py

```diff
@@ -59,10 +59,7 @@ def _create_groupe_role(
             params["authorizedGroups"] = group_block
             return params
 
-        raise RuntimeError(
```

Comment:
maybe add a condition when we have group but not role, or we have role but not group => then raise runtimeError.